examples/widget_demo.py

```diff
@@ -1,10 +1,10 @@
 """Widget demonstration of magicgui."""
```

Comment:
This widget does not have the "calculate" problem that I noted in the snells law gui case. I see output as soon as I click calculate without having to tab away.

Reply:
very interesting!  but also mysterious! :)  i'm stumped (but... still can't repeat the phenotype of the snells_law problem)